joss_qmc/paper.md

```diff
@@ -0,0 +1,114 @@
+---
+title: 'Quasi-Monte Carlo Methods in Python'
+tags:
+  - Python
+  - SciPy
+  - statistics
+  - Quasi-Monte Carlo methods
+authors:
+  - name: Pamphile T. Roy
+    affiliation: 1
+    corresponding: true
+    orcid: 0000-0001-9816-1416
+  - name: Art B. Owen
+    affiliation: 2
+    orcid: 0000-0001-5860-3945
+  - name: Maximilian Balandat
+    affiliation: 3
+    orcid: 0000-0002-8214-8935
+  - name: Matt Haberland
+    affiliation: 4
+    orcid: 0000-0003-4806-3601
+affiliations:
+    - name: Quansight
+      index: 1
+    - name: Stanford University
+      index: 2
+    - name: Meta
+      index: 3
+    - name: California Polytechnic State University, San Luis Obispo, USA
+      index: 4
+date: 30 August 2022
+bibliography: paper.bib
+
+---
+
+# Summary
+
+
+# Statement of need
+
+NumPy random number generators (`numpy.random`) have become the de-facto
+standard for sampling random numbers in the scientific Python ecosystem.
+These methods are fast and reliable, and the results are repeatable when a
+seed is provided. As a foundational tool, NumPy only provides classical
+Monte Carlo (MC) methods. Sampling in high dimensions with MC produces a lot of
+gaps and clusters of points. When these random numbers are used in algorithms
```

Comment:
> As a foundational tool, NumPy only provides classical Monte Carlo (MC) methods.

There are a few things here:

- NumPy doesn't "provide Monte Carlo (MC) methods." It only provides pseudorandom number generators.
- The purpose of the sentence below is to define the term "Monte Carlo" methods and the relationship with pseudorandom numbers, so I'd rather not use the term "Monte Carlo" until that sentence.
- We wouldn't want to suggest that a foundational tool should not provide QMC, right? And SciPy _is_ foundational.

Since the sentence isn't really needed to make the point, let's just remove it.

```suggestion
seed is provided. However, sampling in high dimensions with pseudorandom numbers tends to produce
gaps and clusters of points. When these random numbers are used in algorithms
```